native/src/components/PageDetail.tsx

```diff
@@ -1,27 +1,55 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components/native'
 
+import { InternalPathnameParser } from 'shared'
+
+import buildConfig from '../constants/buildConfig'
 import { contentDirection } from '../constants/contentDirection'
+import useNavigate from '../hooks/useNavigate'
 
-const Identifier = styled.Text`
-  font-family: ${props => props.theme.fonts.native.contentFontBold};
-  color: ${props => props.theme.colors.textColor};
-`
 const DetailContainer = styled.Text<{ language: string }>`
-  display: flex;
   flex-direction: ${props => contentDirection(props.language)};
   font-family: ${props => props.theme.fonts.native.contentFontRegular};
   color: ${props => props.theme.colors.textColor};
 `
 
+const Identifier = styled.Text`
+  font-family: ${props => props.theme.fonts.native.contentFontBold};
+  color: ${props => props.theme.colors.textColor};
+`
+
+const StyledButton = styled.Pressable`
+  /* Offset for bold sibling + underline */
+  margin-top: -2px;
+`
+
+const ButtonText = styled.Text`
+  color: ${props => props.theme.colors.linkColor};
+  text-decoration: underline;
+  text-decoration-color: ${props => props.theme.colors.linkColor};
+`
+
 type PageDetailProps = {
   identifier: string
   information: string
   language: string
+  locationPath?: string | null
 }
 
-const PageDetail = (props: PageDetailProps): ReactElement => {
-  const { identifier, information, language } = props
+const PageDetail = ({ identifier, information, language, locationPath }: PageDetailProps): ReactElement => {
+  const { navigateTo } = useNavigate()
+
+  if (locationPath) {
+    const parsedRoute = new InternalPathnameParser(locationPath, language, buildConfig().featureFlags.fixedCity).route()
+    return (
+      <DetailContainer language={language}>
+        <Identifier>{identifier}: </Identifier>
+        <StyledButton onPress={() => navigateTo(parsedRoute)}>
+          <ButtonText>{information}</ButtonText>
+        </StyledButton>
```

Comment:
:upside_down_face: Personal preference:
```suggestion
        {route ? (
          <StyledButton onPress={() => navigateTo(parsedRoute)}>
            <ButtonText>{information}</ButtonText>
          </StyledButton>
        )  : information}
```